cinema/views.py

```diff
@@ -56,3 +79,51 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
+
+        _date = self.request.query_params.get("date")
+        if _date:
+            year, month, day = (int(_) for _ in _date.split("-"))
+            queryset = queryset.filter(
+                show_time__year=year,
+                show_time__month=month,
+                show_time__day=day
+            )
```

Comment:
read the checklist

Reply:
![image](https://github.com/mate-academy/py-tickets-orders/assets/143685044/49d8258d-12c8-40f3-a102-a91147a6ccae)
![image](https://github.com/mate-academy/py-tickets-orders/assets/143685044/c10ed44a-8c38-40ea-a038-7beeed614c59)


_date is of str type how it could be ==  datetime type?